src/WorldStorage/WSSAnvil.cpp

```diff
@@ -3122,6 +3143,58 @@ bool cWSSAnvil::LoadMonsterBaseFromNBT(cMonster & a_Monster, const cParsedNBT &
 		a_Monster.SetCustomNameAlwaysVisible(CustomNameVisible);
 	}
 
+	// Leashed to a knot
+	bool Leashed;
+	int LeashedIdx = a_NBT.FindChildByName(a_TagIdx, "Leashed");
+	if (LeashedIdx >= 0)
```

Comment:
Perhaps even move this into a separate function, since `LoadMonsterBase` is already pretty long by itself.